File: app/src/main/java/nz/co/warehouseandroidtest/Utils/PreferenceUtil.java
package nz.co.warehouseandroidtest.Utils;

import android.content.Context;
import android.content.SharedPreferences;

import nz.co.warehouseandroidtest.Constants;

public class PreferenceUtil {

    public static String getUserId(Context context) {
        SharedPreferences sharedPreferences = context.getSharedPreferences(Constants.PREF_USER_ID, Context.MODE_PRIVATE);
        return sharedPreferences.getString(Constants.PREF_USER_ID, null);
    }

    public static void putUserId(Context context, String userId) {
        SharedPreferences sharedPreferences = context.getSharedPreferences(Constants.PREF_USER_ID, Context.MODE_PRIVATE);
        SharedPreferences.Editor editor = sharedPreferences.edit();
        editor.putString(Constants.PREF_USER_ID, userId);
        editor.commit();
    }

}

File: app/src/main/java/nz/co/warehouseandroidtest/utils/PreferenceUtil.kt
package nz.co.warehouseandroidtest.utils

import android.content.Context

object PreferenceUtil {
    fun getUserId(context: Context): String? {
        val sharedPreferences =
            context.getSharedPreferences(Constants.PREF_USER_ID, Context.MODE_PRIVATE)
        return sharedPreferences.getString(Constants.PREF_USER_ID, null)
    }

    fun putUserId(context: Context, userId: String) {
        val sharedPreferences =
            context.getSharedPreferences(Constants.PREF_USER_ID, Context.MODE_PRIVATE)
        sharedPreferences.edit().run {
            putString(Constants.PREF_USER_ID, userId)
            commit()
        }
    }
}